src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,46 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
```

Comment:
you can also make the interface in a separate file
 ` export interface Movie {
  title: string,
  description?: string,
  imgUrl:string,
  imdbUrl: string,
  imdbId: string,
}`

and after it write 
`type Props = {
  movie: Movie;
};
`